connectors/sources/zoom.py

```diff
@@ -341,6 +341,12 @@ def get_default_configuration(cls):
                 "sensitive": True,
                 "type": "str",
             },
+            "recording_age": {
+                "label": "Recording Age Limit (Months)",
+                "order": 5,
+                "tooltip": "Set the maximum age (in months) for retaining recordings.",
```

Comment:
This isn't about _retaining_ recordings (zoom is what's _retaining_ things), but about how far back in history to search. For slack, we worded this like:
```
            "fetch_last_n_days": {
                "display": "numeric",
                "label": "Days of message history to fetch",
                "order": 2,
                "tooltip": "How far back in time to request message history from slack. Messages older than this will not be indexed.",
                "type": "int",
            },
```

Could we do something similar here?

Reply:
Sure!